src/stories/tasks/TasksTable.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Tasks/Table/Intro" />
+
+# TasksTable
+
+**Shows a list of tasks within an epic.**
+
+## Overview
+
+The **TasksTable** component is an
+[SLDS DataTable](https://react.lightningdesignsystem.com/components/data-tables/)
+that shows a list of **Tasks** and their respective current statuses within an
+epic.
+
+<Story id="tasks-table-component--component" />
+
+## Sub-components
+
+### Columns & Cells
+
+**Tasks**
+
+A description of the **Task** that links to its details page.
+
+**Status**
+
+Displays the **Task**'s status as `Planned`, `In Progress`, `Complete`,
+`Changes Requested`, or `Approved`.
+
```

Comment:
@stacyk Yeah, I think that'd be good to mention. If an "In Progress" Task has a valid review (that is, a review was submitted and no new commits have been made since then), then it is either "Changes Requested" or "Approved". Otherwise we display the task status.

Reply:
How's this? @jgerigmeyer @stacyk 
![image](https://user-images.githubusercontent.com/41588129/106164587-6c94bc00-618a-11eb-834b-b32333d14461.png)

Reply:
@dvdherron I wonder if this sounds like you will see different status' based on your role. Wordsmith/format as you see fit:

- Tasks are marked as `Planned` if there have not been any retrieved (that word sounds weird in this context) changes, or commits to the task.
- Once the task has at least 1 retrieved change it will display a status of `In Progress`.
- After the task has been submitted for testing, and the tester responds, the status becomes either `Changes Requested` or `Approved` depending on their feedback.
- Once all changes have been made and accepted (the branch can be merged?) the status becomes `Complete`. 

@jgerigmeyer can you confirm this last one (or any)?

Reply:
@stacyk @dvdherron That's close. Here's the logic:

- A task is "Planned" until a scratch org has been created, then it becomes "In Progress"
- It remains "In Progress" until a tester submits review, at which point it becomes "Changes Requested" or "Approved"
- If changes are made/retrieved (even after review), it goes back to "In Progress"
- Once the task PR has been merged on GitHub, it becomes "Complete"

Reply:
Remixed what you wrote about order and how statuses change, @jgerigmeyer . Have another look to make sure this is clear, @stacyk.

Reply:
@dvdherron This is great! Thanks for adding that clarity.